server/catgenome/src/main/java/com/epam/catgenome/controller/vo/externaldb/NCBITaxonomyVO.java

```diff
@@ -41,12 +41,27 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class NCBITaxonomyVO {
 
+    @JsonProperty(value = "taxid")
+    private String taxid;
+
     @JsonProperty(value = "scientificname")
     private String scientificname;
```

Comment:
Since we are using `JsonProperty` with original field name, let's use standard camel case for field names, e.g. `scientificName`